### PR TITLE
edit links to dropapp mobile views

### PR DESCRIPTION
**Pull request by HaGuesto**
Reason:
- set preference Session variable in dropapp to classic
- go to the same menu in dropapp on mobile.

### CODE REVIEW

File: react/src/utils/helpers.ts [hunk 1]
```diff
@@ -1,25 +1,50 @@
...
-}
+};
+
+export const generateDropappUrl = (
+  oldLink: String,
+  baseId: String | undefined,
+  qrCode: String | undefined,
+  labelIdentifier: String | undefined,
+) => {
+  const newLink = "https://" + oldLink + "?camp=" + baseId + "&preference=classic";
```
Comment (pylipp) [inline react/src/utils/helpers.ts:41]: `oldLink` already contains the protocol prefix, so we don't have to specify it another time. See [this comment](https://github.com/boxwise/boxtribute/pull/466#discussion_r1027113171). Also it then works when running locally (for `REACT_APP_OLD_APP_BASE_URL` I have set `http://localhost:8100`; if https:// is added the link becomes mixed up)

**Review by pylipp — CHANGES_REQUESTED** [review]
The links work locally when removing the https:// prefix (see comment). Why is the precise redirecting for mobile only?
Can you link a Trello card?

File: react/src/utils/helpers.ts (hunk 1 above)
Comment (HaGuesto) [reply react/src/utils/helpers.ts:41]: I removed the protocol.

**Comment by HaGuesto** [comment]
> Why is the precise redirecting for mobile only? 
Only for the menuItem "Go to classic mobile" corresponding views exist in dropapp. So, only then should a different link then the homepage be created.

**Comment by HaGuesto** [comment]
[2.0 iteration on forwarding to dropapp](https://trello.com/c/ZDlfoF7n/1048-20-iteration-on-forwarding-to-dropapp)

**Review by pylipp — APPROVED** [review]

**Pull request merged**